CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ All notable changes to this project will be documented in this file.
 * ### `nidcpower` (NI-DCPower)
     * #### Added
     * #### Changed
+        * The byte packing alignment of `struct_NILCRLoadCompensationSpot` is changed from 4-byte to 8-byte in Linux.
```

Comment:
This needs to say why and what to do about it or what effects it has on customer code.

> The byte packing alignment of `struct_NILCRLoadCompensationSpot` is changed from 4-byte to 8-byte in Linux in order to match the driver runtime for NI-DCPower 2023 Q1.

Reply:
I have updated the CHANGELOG statement based on [your other suggestion](https://github.com/ni/nimi-python/pull/1870#discussion_r1038439451), kindly let me know if you want both of them to be included.